src/scales/scale.logarithmic.js

```diff
@@ -160,52 +144,25 @@ class LogarithmicScale extends Scale {
 		return this.getPixelForValue(ticks[index].value);
 	}
 
-	/**
-	 * Returns the value of the first tick.
-	 * @param {number} value - The minimum not zero value.
-	 * @return {number} The first tick value.
-	 * @private
-	 */
-	_getFirstTickValue(value) {
-		const exp = Math.floor(log10(value));
-		const significand = Math.floor(value / Math.pow(10, exp));
-
-		return significand * Math.pow(10, exp);
-	}
-
 	_configure() {
 		const me = this;
 		let start = me.min;
-		let offset = 0;
 
 		Scale.prototype._configure.call(me);
 
-		if (start === 0) {
-			start = me._getFirstTickValue(me.minNotZero);
-			offset = valueOrDefault(me.options.ticks.fontSize, defaults.fontSize) / me._length;
-		}
-
 		me._startValue = log10(start);
-		me._valueOffset = offset;
-		me._valueRange = (log10(me.max) - log10(start)) / (1 - offset);
+		me._valueRange = log10(me.max) - log10(start);
 	}
 
 	getPixelForValue(value) {
 		const me = this;
-		let decimal = 0;
-
-		if (value > me.min && value > 0) {
-			decimal = (log10(value) - me._startValue) / me._valueRange + me._valueOffset;
-		}
-		return me.getPixelForDecimal(decimal);
+		return me.getPixelForDecimal((log10(value || me.min) - me._startValue) / me._valueRange);
```

Comment:
It seems unexpected to me to do `value || me.min`. If we have `!value` then can we just return `null` or `NaN` or something from this `function`?

Reply:
That would change the `0` case too, not sure we want to do that?

Reply:
Ok, changed it so `0` will be parsed as `null`, other invalid values `NaN`. Defaulting `null` to `me.min` now, `NaN` will be `NaN`.